output/theme.sasslint.txt

```diff
@@ -2,11 +2,14 @@
 
 
 src/theme/_buttons.scss
-  14:2  error  The keyword `all` should not be used with the property `transition`  no-transition-all
+  16:2  error  The keyword `all` should not be used with the property `transition`  no-transition-all
+
+src/theme/_colors.scss
+  37:13  error  Hex values should use the shorthand format - 3 characters where possible  hex-length
```

Comment:
I reverted the color change, so this linter error should be removed in next ci run